src/Conversations/ConversationInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Telegram\Bot\Conversations;
+
+use Telegram\Bot\Api;
+use Telegram\Bot\Objects\Update;
+
+interface ConversationInterface
+{
+    /**
+     * Process Inbound Non-Command Message.
+     *
+     * @param Api $telegram
+     * @param Update $update
+     *
+     * @return mixed
+     */
+    public function make(Api $telegram, Update $update);
+
+    /**
+     * Handle Conversation.
+     *
+     * @return mixed
+     */
+    public function handle();
```

Comment:
I think this method is redundant. It only used in AbstractCommand and should not be public